src/components/DateInput/DateInput.js

```diff
@@ -84,7 +85,7 @@ const PrivateDateInput = (props) => {
   })
 
   const getClasses = () => {
-    const base = `DateInput ${styles.TextInputCommon}`
+    const base = fullstoryMask ? `DateInput ${styles.TextInputCommon} fs-exclude` : `DateInput ${styles.TextInputCommon}`
```

Comment:
https://help.fullstory.com/hc/en-us/articles/360020623574-How-do-I-protect-my-users-privacy-in-Fullstory#01F5DPW1AJMP73R81PE54AENFE